tutorials/image/cifar10_estimator/generate_cifar10_tfrecords.py

```diff
@@ -81,6 +80,7 @@ def convert_to_tfrecord(input_files, output_file):
       record_writer.write(example.SerializeToString())
   record_writer.close()
 
+
 def main(argv):
   del argv  # Unused.
```

Comment:
Typically I've seen this argument called `unused_argv` without any lines that use `del`.